vendor/code.cloudfoundry.org/clock/clock.go

```diff
@@ -29,7 +29,7 @@ func (clock *realClock) Now() time.Time {
 }
 
 func (clock *realClock) Since(t time.Time) time.Duration {
-	return time.Now().Sub(t)
+	return time.Since(t)
 }
```

Comment:
This, and a small change in the file below, look to be the only actual changes (everything else is dependabot-forced updates of `golang.org/x/tools`) "yay".